src/components/new-request/search.vue

```diff
@@ -657,6 +747,10 @@ export default class Search extends Mixins(CommonMixin, NrAffiliationMixin) {
 
     this.setLocation(jurisdiction.group === 0 ? Location.CA : Location.IN)
     this.setJurisdictionCd(jurisdiction.value)
+    // Prevent entity type from being cleared when jurisdiction is changed
+    if (this.business) {
+      this.setEntityTypeCd(this.business.legalType)
```

Comment:
When user changes jurisdiction (from Manitoba to United States for example), the entity type was being cleared. In doing that, the check your name button was not being shown (everything is so tangled together).

Reply:
Yes, very. I'm trying for us to be careful to not extend this pattern.

Can you tell what code cleared the legal type, and maybe pre-empt that?

Reply:
The code that's clearing it is line 744: `this.setLocation(jurisdiction.group === 0 ? Location.CA : Location.IN)`.

However, I'd prefer to leave it as is Sev in case other flows might need it to re-initialize/re-populate some of the variables. Please let me know if that's not OK with you Sev.

Reply:
OK but please change your comment -- it's not "preventing" anything; it's resetting the entity type code etc.

Reply:
Thanks.